session/mock_bootstrap.go

```diff
@@ -107,7 +108,12 @@ func mockUpgradeToVerLatest(s Session, ver int64) {
 		TestHook.OnBootstrap(s)
 		mustExecute(s, sql)
 		logutil.BgLogger().Info("mock upgrade exec", zap.String("sql", sql))
-		time.Sleep(20 * time.Millisecond)
+		// Make test faster.
+		if intest.InTest {
+			time.Sleep(5 * time.Millisecond)
+		} else {
+			time.Sleep(30 * time.Millisecond)
+		}
```

Comment:
I think the whole function `mockUpgradeToVerLatest` is only used in tests?

Reply:
No，we will use it in IT

Reply:
Could you please update the comment to indicate that the new one is for unit test?